exercises/dnd-character/description.md

```diff
@@ -0,0 +1,30 @@
+For a game of Dungeons & Dragons, you generate your character's six ability
```

Comment:
Maybe also introduce the concept of a character, listing the various properties (strength, etc.).

Reply:
OK.